### ISSUE TITLE
 output all possible combination?

### ISSUE THREAD

**Issue by maddo7** [melnaron/mel-spintax#1]
Is it possible to output an array with all possible spintax combinations?

**Comment by melnaron** [comment]
No, there is no such functionality at the moment, but it's relatively easy to add it. There should be a new method similar to `unspin` that instead of getting you random match will recursively go through all the combinations and add them all into an array that will be returned on the end of operation.
